pyhomematic/_hm.py

```diff
@@ -282,9 +282,15 @@ def listDevices(self, interface_id):
             self._devices_raw[remote] = []
         if self.systemcallback:
             self.systemcallback('listDevices', interface_id)
-        return self._devices_raw[remote]
 
-    def newDevices(self, interface_id, dev_descriptions):
+
+        if len(self._devices_raw[remote]) > 2:
```

Comment:
The value of `2` seems very specific. Where does this come from? It's been a while since I worked with this code, but I somehow have the feeling this would impact setups from users who use more than just HmIPW. Do you have _regular_ devices as well and can verify, that they keep being detected properly?

Reply:
It was actually derived from experiements. I guess the safest way would be the fitting to the 8192 bytes boundary. Anyhow, eventually its not mandatory to return an filled array at all. So, if you feel uneasy with this more explorative setting, shall an empty array returned?

Reply:
Actually, this __should__ return the actual list of devices and not just an empty array. The empty array would fix the bug we are seeing specifically for HmIP. This bug will be fixed in the upcoming firmware update for the CCU / RaspberryMatic as mentioned here: https://github.com/jens-maus/RaspberryMatic/issues/843
So we don't really need this workaround in the long run. Since currently HmIPW isn't working anyway I would suggest du delay this PR until the new firmware is available and then see if workarounds are still required.

Reply:
Understood for the long run. Just not clear when it comes and if it really solves both issues the initdevice/generic error AND the listdevice/8192 limitation. As in my case now the complete 2010 proxy.init crashes, mandatory elements like covers are not working. Also I was not clear on the merrit when returning the device list and did not see a downside on this fix. Hence, I wanted to publish this work around in case other people run into the same situation. You may decide also based on the your insights into the fix time line.